app/src/main/java/br/com/dionataferraz/vendas/account/usecase/updateAccountUseCase.kt

```diff
@@ -0,0 +1,40 @@
+package br.com.dionataferraz.vendas.account.usecase
+
+import br.com.dionataferraz.vendas.App
+import br.com.dionataferraz.vendas.account.data.local.AccountDatabase
+import br.com.dionataferraz.vendas.account.data.local.AccountEntity
+import br.com.dionataferraz.vendas.account.data.local.Operation
+import br.com.dionataferraz.vendas.transaction.data.TransactionEntity
+import br.com.dionataferraz.vendas.transaction.usecase.InsertTransactionUsecase
+import java.util.*
+
+class updateAccountUseCase(private val acc: AccountEntity,
+                           private val operation: Operation,
+                           private val amount: Double) {
```

Comment:
Padrão de formatação
```suggestion
class UpdateAccountUseCase(
                           private val acc: AccountEntity,
                           private val operation: Operation,
                           private val amount: Double
) {
```